tasks/certificates.yml

```diff
@@ -114,5 +114,5 @@
 - name: sign certificate request on master
   command: >
     icinga2 ca sign {{ icinga2_client_register_certfinger['stdout'] }}
-  delegate_to: "vm-adsy-monitoring-prod-westeurope-master-1.adfinis-sygroup.ch"
+  delegate_to: "{{ icinga2_client_csr_signing_master | default('vm-adsy-monitoring-prod-westeurope-master-1.adfinis-sygroup.ch') }}"
```

Comment:
Hi @flybyray 
Thank you very much for your contribution! Nice find here. Please remove the `default()`, I've already fixed this in our internal playbook and the change shouldn't break anything on our or other people using the role :)